src/main/java/io/jenkins/plugins/opentelemetry/job/log/OtelLogSenderRoot.java

```diff
@@ -0,0 +1,13 @@
+package io.jenkins.plugins.opentelemetry.job.log;
+
+/**
+ * Sender for logs send from the controller, pipeline flow control and pipeline steps ran in Jenkins controller.
+ * See https://github.com/jenkinsci/pipeline-cloudwatch-logs-plugin/blob/master/src/main/java/io/jenkins/plugins/pipeline_cloudwatch_logs/CloudWatchSender.java#L79
+ */
+final class OtelLogSenderRoot extends OtelLogSender {
```

Comment:
I simplified just after our call removing OtelLogSenderRoot and OtelLogSenderFlowNode just keep one OtelLogSenderBuildListener, sorry for the mess